wiki/Tournaments/GBC/GBC_2022_Spring/en.md

```diff
@@ -0,0 +1,208 @@
+
+1. A referee will create a multiplayer room 15 minutes in advance. Players must gather during this period.
+   - Room settings are osu!mania, Team-Vs., Win Condition: 'ScoreV2'. Room name must be "GBC2022Spring: (TeamRed) vs (TeamBlue)".
+   - The team mentioned first in the room name must be the red team, the team mentioned second in the room name must be the blue team.
+2. Each captain can ban **one beatmap** to be selected from the pool. These beatmaps can not be picked by any team in the entire match.
+3. Beatmap selection will alternate between each captain selecting a beatmap out of the mappool.
+4. Each captain must use `!roll` once in `#multiplayer`.
+   - The winner of the `!roll` starts picking the first beatmap of the match.
+   - The loser of the `!roll` starts banning one beatmap, followed by the winner of the `!roll` to ban a beatmap. 
+5. After each beatmap selected, captain could decide exclusion one specific opposite player for selected beatmap in this match, except tie-breaker.
+   - Each player will only be exclusion only **once** per match.
+   - captain have rights not to exclusion any player in this beatmap.
+6. Teams will have 2 minutes to ban or pick a beatmap , 2 minutes to exclusion opposite player ,and 3 minutes to get ready. If a team takes more time than alloted, the procedures adopted will be as follows:
+   - For the first occurrence:
+     - The team will receive a verbal warning from the referee
+   - On subsequent occurrences:
+     - For exclusion player timer: See as captain forfeit to exclusion player.
+     - For a map ban timer:  See as captain forfeit to ban a beatmap.
+     - For a map pick timer: a random map will be chosen from the mappool using `!roll X`, where X is the number of maps that were neither picked nor banned, excluding the Tiebreaker.
+     - For a ready timer: the referee will issue the `!mp start 10` command, regardless of how many players from each team are present in the lobby, using `!mp kick` on any extra players for each team, starting from the top (i.e. the first valid player combination for each team will be forced to play the pick). The results for such games are to be taken as is.
+   - Repeat offenders may receive further sanctions from the Host
+   - Each team will receive **one** "tactical timeout" of **two minutes**, to be used as extra time to pick or ban a beatmap. *The tactical timeout is optional, and is not required to be used*. 
+7. Referee reserves the right to modify these rules or decided which team violation rules for mataches they responsible for. This can only happen if necessary.
+
+### Mappool instructions
+
+1. There will be one mappool for each stage.
+   - The whole Regular stage will be seen as same stage, that means a  mappools will be used in all match in Regular stage.
+2. Each mappool consists of a fixed amount of maps each stage which will all be played under FreeMod conditions. This means that there is a unique FreeMod bracket.
```

Comment:
unless this means that every mappool has the same freemod bracket, this point is also very redundant and doesn't serve much purpose

```suggestion
2. The FreeMod bracket will be played under FreeMod conditions.
```

Reply:
yes, it's a rule reference from MWC rules. I'm wondering why and think maybe it used for differ from other game mode?
so it will kept as it was.

Reply:
can keep for now